src/vm/types/execution_scopes.zig

```diff
@@ -5,9 +5,27 @@ const Felt252 = @import("../../math/fields/starknet.zig").Felt252;
 const HintError = @import("../error.zig").HintError;
 
 pub const HintType = union(enum) {
+    const Self = @This();
     // TODO: Add missing types
     felt: Felt252,
     u64: u64,
+    felt_map_of_arr_u64: std.AutoHashMap(Felt252, std.ArrayList(u64)),
+    arr_u64: std.ArrayList(u64),
```

Comment:
In order to be consistent with https://github.com/keep-starknet-strange/ziggy-starkdust/pull/442, can we call this one `u64_list`?